superflore/generators/ebuild/metadata_xml.py

```diff
@@ -28,7 +28,8 @@ def get_metadata_text(self):
         ret += "<!DOCTYPE pkgmetadata SYSTEM "
         ret += "\"http://www.gentoo.org/dtd/metadata.dtd\">\n"
         ret += "<pkgmetadata>\n"
-        if self.longdescription is not None:
+        if self.longdescription is not None and\
```

Comment:
In python `is not None` is redundant with just `if args.distro:` it will also catch other zero values such as an empty string or list. https://stackoverflow.com/questions/7816363/if-a-vs-if-a-is-not-none

Usually you only want to use the very specific meaning of None if you want to differentiate between other potential zero values.